MyCryptoHQ/MyCrypto
# Removed etc - epool node

Closes #2341 

### Description

Removed epool.io etc node at the request of epool

### Changes

* Removed ETC - epool.io network at the request of epool.

### Steps to Test

1. Click the `Network` dropdown in the top.
2. Select the `Ethereum Classic` Network. Make sure that epool.io is not showing as one of the available nodes.

### Screenshots

![remove epool](https://user-images.githubusercontent.com/29407814/50739961-32c47780-11b5-11e9-91e8-50eb9b80361b.PNG)
